include/zephyr/audio/dmic.h

```diff
@@ -43,110 +43,129 @@ extern "C" {
  * DMIC driver states
  */
 enum dmic_state {
-	DMIC_STATE_UNINIT,	/* Uninitialized */
-	DMIC_STATE_INITIALIZED,	/* Initialized */
-	DMIC_STATE_CONFIGURED,	/* Configured */
-	DMIC_STATE_ACTIVE,	/* Active */
-	DMIC_STATE_PAUSED,	/* Paused */
+	DMIC_STATE_UNINIT,	/**< Uninitialized */
+	DMIC_STATE_INITIALIZED,	/**< Initialized */
+	DMIC_STATE_CONFIGURED,	/**< Configured */
+	DMIC_STATE_ACTIVE,	/**< Active */
+	DMIC_STATE_PAUSED,	/**< Paused */
 };
 
 /**
  * DMIC driver trigger commands
  */
 enum dmic_trigger {
-	DMIC_TRIGGER_STOP,	/* stop stream */
-	DMIC_TRIGGER_START,	/* start stream */
-	DMIC_TRIGGER_PAUSE,	/* pause the stream */
-	DMIC_TRIGGER_RELEASE,	/* release paused stream */
-	DMIC_TRIGGER_RESET,	/* reset */
+	DMIC_TRIGGER_STOP,	/**< Stop stream */
+	DMIC_TRIGGER_START,	/**< Start stream */
+	DMIC_TRIGGER_PAUSE,	/**< Pause stream */
+	DMIC_TRIGGER_RELEASE,	/**< Release paused stream */
+	DMIC_TRIGGER_RESET,	/**< Reset stream */
 };
 
 /**
  * PDM Channels LEFT / RIGHT
  */
 enum pdm_lr {
-	PDM_CHAN_LEFT,
-	PDM_CHAN_RIGHT,
+	PDM_CHAN_LEFT,	/**< Left channel */
+	PDM_CHAN_RIGHT, /**< Right channel */
```

Comment:
Same comments about `enum` as in #60362,  otherwise looks good, thx.

Reply:
I understand, but on the other hand I would like to point out, as you can see in the diff, that most of the other enums already had these kind of seemingly redundant comments :)
Are you able to approve?